css/contact.css

```diff
@@ -0,0 +1,131 @@
+#contact-button {
+  display: flex;
+  align-items: center;
+  border-radius: 8px;
+  padding: 20px;
+  position: relative;
+  top: 140px;
+  width: 150px;
+  height: 30px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  border: none;
+}
+
+#contact-button:hover {
+  background-color: #2230d2;
+  color: #fff;
+}
+
+#contact {
+  background-image: url(/Images/contact\ form\ shapes-mobile@2x.png);
+  background-color: #6070ff;
+  background-repeat: no-repeat;
+  background-position-x: 120px;
+  background-position-y: 12px;
+  background-size: auto 757px;
+  border-radius: 100px 0% 0% 0%;
+  padding: 20px;
+  height: 773px;
+  position: relative;
+  bottom: 15vh;
+  display: flex;
+  flex-flow: column wrap;
+  width: 327px;
+  margin-top: 10vh;
+}
+
+#contact-background {
+  position: relative;
+  left: 30%;
+  height: 30vh;
+}
+
+form {
+  order: 3;
+  position: relative;
+  bottom: 50px;
+}
+
+#contact p {
+  position: relative;
+  width: 327px;
+  height: 102px;
+  top: 60px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #ebebff;
+}
+
+#contact h2 {
+  position: relative;
+  width: 100%;
+  height: 56px;
+  left: 5%;
+  top: 30px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
```

Comment:
- [ ] Great job fixing the submit button, now we can see the form submission in action, and a great job working on your horizontal scroll, I can clearly see the difference. But there is still a little bit of horizontal scrolling happening. Please make sure to fix it and make sure your sections take up the fullscreen to make it look like the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=wgGEhimPzdFHgsEn-0) as it is the [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#project-requirements) to stick to the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=wgGEhimPzdFHgsEn-0)

> You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

https://user-images.githubusercontent.com/44798044/220824780-225115bf-e08d-4527-9e9c-124e1275d635.mov